File: redsheet-portal/src/app/database-models/v-modulewise-content.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vModulewiseContent {
    constructor(vModulewiseContent?: vModulewiseContent )  {
        let properties = [ "action", "afar", "applicationModuleId", "english", "french", "languageContentId", "languageContentName", "languageContentType", "moduleContentId", "moduleMasterId", "moduleMasterName",];
        for (let property of properties)
            if (vModulewiseContent && vModulewiseContent[property])
                this[property] = vModulewiseContent[property];
    }
 
    @required()
    @maxLength(10)
	action : string =   undefined;
 
	afar : string =   undefined;
 
	applicationModuleId : number =   0 ;
 
	english : string =   undefined;
 
	french : string =   undefined;
 
    @range(1,2147483647)
	languageContentId : number =   undefined;
 
    @required()
    @maxLength(50)
	languageContentName : string =   undefined;
 
    @maxLength(20)
	languageContentType : string =   undefined;
 
    @range(1,2147483647)
	moduleContentId : number =   undefined;
 
    @range(1,2147483647)
	moduleMasterId : number =   undefined;
 
    @required()
    @maxLength(100)
	moduleMasterName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vModulewiseContent.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vModulewiseContents", Schema = "dbo")]
	[Serializable]
	public partial class vModulewiseContent  
	{
 
	[Required] 
	[MaxLength(10)]
	public string Action { get; set; } 

	public string Afar { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int ApplicationModuleId { get; set; } 

	public string English { get; set; } 

	public string French { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int LanguageContentId { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string LanguageContentName { get; set; } 
 
	[MaxLength(20)]
	public string LanguageContentType { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ModuleContentId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ModuleMasterId { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string ModuleMasterName { get; set; } 

	public vModulewiseContent()
	{
    }

    }
}

